example/brunel/brunel_miniapp.cpp

```diff
@@ -304,12 +293,12 @@ int main(int argc, char** argv) {
     return 0;
 }
 
-void banner(hw::node_info nd, const distributed_context* ctx) {
+void banner(domain_info nd, const distributed_context* ctx) {
```

Comment:
Yeah, I reiterate that `domain_info` is not a name I like. `node_info` was more descriptive, but not great either.
`domain_resources`, `local_resources`, `hw_resources`, `local_hw`... ?

Reply:
So, we're splitting a problem into subdomains, and assign each domain to _thing_, which is an element of our set of distributed processes. Quick gander on the web seems to indicate that everyone just calls these 'processes', and so we need a name for the information used to describe the configuration and resources that a process should use to run its part of the problem.

I don't want to use `hw` as the hardware operates at a different, lower level of abstraction. 'Local' describes where these resources come from, but not what they are.

How about one of:
* `process_resource` (or shorter variations, e.g. `proc_res`, `process_res`, `proc_resource`)
* `process_config` (or shorter variations).

Reply:
`proc_resources`?

Reply:
Conclusion from pers. comm: `proc_allocation`